src/wallet/wallet.ts

```diff
@@ -163,6 +164,48 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     // TODO should we have a debug mode?
   }
 
+  /**
+   * Sets the server to connect on config singleton and storage
+   *
+   * @param {String} newSever - The new server to set the config and storage to
+   *
+   * @memberof HathorWalletServiceWallet
+   * @inner
+   */
+  changeServer(newServer: string) {
+    storage.setItem('wallet:wallet_service:base_server', newServer);
+    config.setWalletServiceBaseUrl(newServer);
+  }
+
+  /**
+   * Sets the websocket server to connect on config singleton and storage
+   *
+   * @param {String} newSever - The new websocket server to set the config and storage to
+   *
+   * @memberof HathorWalletServiceWallet
+   * @inner
+   */
+  changeWsServer(newServer: string) {
+    storage.setItem('wallet:wallet_service:ws_server', newServer);
+    config.setWalletServiceBaseWsUrl(newServer);
+  }
+
+  /**
+   * Gets the stored websocket and base server urls
+   *
+   * @memberof HathorWalletServiceWallet
+   * @inner
+   */
+  static getServerUrlsFromStorage(): WalletServiceServerUrls {
+    const walletServiceBaseUrl = storage.getItem('wallet:wallet_service:base_server');
+    const walletServiceWsUrl = storage.getItem('wallet:wallet_service:ws_server');
```

Comment:
These informations have a similar storage as the 'wallet:network': they are both on the config and on the storage.

Won't we have here also the priority algorithm that we have on `src/config.ts`:`getNetwork()`?

Reply:
That's a valid point, but since the method is called getServerUrls**FromStorage**, I think we should always get it from `storage`, even if the config singleton has a different value on it — maybe the user is rolling back to the stored value after a `config.set`

What do you think?

Reply:
With your explanation it seems ok for me.
Maybe in the future a diagram could explain visually when to use the storage and when to use the singleton for a newcoming developer.